Crypto/Alchemy/Interpreter/PT2CT.hs

```diff
@@ -70,9 +70,7 @@ newtype PT2CT
 -- variance over \( R^\vee \) is \( r / \sqrt{\varphi(m')} \).)
 pt2ct :: forall m'map zqs gad z a ctex e mon .
       -- this forall is for use with TypeApplications at the top level
-  (MonadAccumulator Keys mon, MonadAccumulator Hints mon,
-   MonadReader Double mon) =>
-  PT2CT m'map zqs gad z ctex mon e a
+  (KeysNoiseCtx Double mon) => PT2CT m'map zqs gad z ctex mon e a
```

Comment:
Worth making `Double` into an argument `v` now that you've gone through the trouble? Note that `v` is a fundep of `mon` so even though `v` doesn't appear on the RHS of `=>`, it might be ok.

Reply:
I forgot that `Monad*` used fundeps! I was concerned about adding yet another potentially ambiguous type, but you're right I don't think this a concern thanks to fundeps